### DISCUSSION TITLE
Feedback on my Hackathon Project

### DISCUSSION THREAD

**Discussion by induviduality** [EddieHubCommunity/support#1005] [Q&A]
### Hey everyone! 👋🏻 

I've just submitted a project of mine to AngelHacks today. It is a dice🎲 simulation program using Quantum Computers. It's quite a simple one, but it's my first time exploring Quantum Computing and I'm super happy to make this one. 

I've used the Qiskit SDK and Python to build this project within IBM Quantum Experience. I'd love it if you could check out [my project](https://devpost.com/software/quantum-dice?ref_content=user-portfolio&ref_feature=in_progress) give me feedback on what I should be improving! :)

This is my first time building a relatively large coding project. It was quite challenging but I'm proud to see it through to the end. Please do check my Devpost and let me know what I can do better with regards to anything! Do drop any suggestions/ideas to experiment with on Quantum Computers. I hope to receive some tips that I can immediately use in my next project.

I'd also really appreciate it if you could help me out and vote for [my project on Devpost](https://devpost.com/software/quantum-dice?ref_content=user-portfolio&ref_feature=in_progress) as well. 

[Here's my Github Repo for the project.](https://github.com/indujaaaa/quantum-dice-hack)

Do check it out and give me your insights.

Thanks for your time! 😄

**Comment by eddiejaoude** [comment]
Thanks for sharing @indujaaaa 👍 

Looks great 🤓 , some suggestions on the open source repo
- add a code of conduct
- add a license
- add a contributing file
- I would recommend using the Issues too
